### DISCUSSION TITLE
Could not find org.littleshoot:littleproxy:1.1.3-KFIX.

### DISCUSSION THREAD

**Discussion by LexieAlreadyTaken** [mamoe/mirai#1324] [Q&A] [answered]
在gradle同步的时候突然出现了这个问题。

2.6.6也有这个问题，2.5.2也有这个问题。

报错全文：

  Could not find org.littleshoot:littleproxy:1.1.3-KFIX.
  Required by:
    project : > net.mamoe:mirai-login-solver-selenium:1.0-dev-16

  Possible solution:
  Declare repository providing the artifact, see the documentation at 
  https://docs.gradle.org/current/userguide/declaring_repositories.html

然而这个LittleProxy貌似只出到了1.1.2……这个1.1.3是哪里来的……

**Comment by Karlatemp** [comment, marked as answer]
```
Required by:
project : > net.mamoe:mirai-login-solver-selenium:1.0-dev-16
```
这不很明显告诉你咋了

**Reply by LexieAlreadyTaken** [reply]
dependencies {
    val miraiVersion = "2.5.2"

    // 开发时使用 mirai-core-api，运行时提供 mirai-core
    api("net.mamoe:mirai-core-api:$miraiVersion")
    runtimeOnly("net.mamoe:mirai-core:$miraiVersion")

    runtimeOnly("net.mamoe:mirai-login-solver-selenium:1.0-dev-16")
    compile("org.json:json:20190722")
    api("org.jetbrains.kotlin:kotlin-nosql-mongodb:0.1-SNAPSHOT")
    api("net.mamoe:mirai-login-solver-selenium:1.0-dev-16")

    // 可以简单地只添加 api("net.mamoe:mirai-core:2.0-M1")
}

是这样加吗？我加了还报一样的错（抱歉不太懂Kotlin）

**Reply by Karlatemp** [reply]
Delete it

**Reply by LexieAlreadyTaken** [reply]
噢噢抱歉，我傻了
